samples/highcharts/series/legend-symbol-color/demo.js

```diff
@@ -0,0 +1,52 @@
+(async () => {
+    const data = await fetch(
+        'https://www.highcharts.com/samples/data/range.json'
+    ).then(response => response.json());
+
+    Highcharts.chart('container', {
+        chart: {
+            type: 'arearange',
+            zooming: {
+                type: 'x'
+            },
+            scrollablePlotArea: {
+                minWidth: 600,
+                scrollPositionX: 1
+            }
+        },
+        title: {
+            text: 'Temperature variation by day'
+        },
+        xAxis: {
+            type: 'datetime'
+        },
+        yAxis: {
+            title: {
+                text: ''
+            }
+        },
+        tooltip: {
+            crosshairs: true,
+            shared: true,
+            valueSuffix: '°C',
+            xDateFormat: '%A, %b %e'
+        },
+        series: [{
+            name: 'Temperatures',
+            data: data,
+            color: {
+                linearGradient: {
+                    x1: 0,
+                    x2: 0,
+                    y1: 0,
+                    y2: 1
+                },
+                stops: [
+                    [0, '#ff0000'],
+                    [1, '#0000ff']
+                ]
+            },
+            legendSymbolColor: '#880088' // change legend symbol color
```

Comment:
On the one hand, the idea is ok, on the other hand, this color is not so different from the original gradient that it is hard to see at first glance what has really changed. Personally, I would replace it with some contrasting one, but it is not something I would insist on.

If we were to leave this color, I would simplify the sample by removing `scrollablePlotArea` and other irrelevant options like `crosshair`, etc. 

Anyway, it's nitpicking, so I'm approving it and I'll leave the final judgment to Torstein.